browser/translate/brave_translate_prefs_migration.cc

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/translate/brave_translate_prefs_migration.h"
+
+#include "brave/components/translate/core/common/brave_translate_features.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "components/prefs/pref_service.h"
+#include "components/translate/core/browser/translate_pref_names.h"
+
+namespace translate {
+
+namespace prefs {
+const char kMigratedToInternalTranslation[] =
+    "translate_migrated_to_internal_translation";
```

Comment:
there is a vague convention to prefix our preferences with "brave." :)

Reply:
also not super clear what is an internal translation - maybe to "cloud translation"?

Reply:
Renamed to `brave.translate_migrated_from_extension`